cli/src/pcluster/models/imagebuilder.py

```diff
@@ -78,20 +113,41 @@ def __init__(self, stack_data: dict):
         except AWSClientError:
             self._image_resource = None
 
-    @property
-    def s3_bucket_name(self):
-        """Return the name of the bucket used to store image information."""
-        return self._get_tag(PCLUSTER_S3_BUCKET_TAG)
-
     @property
     def s3_artifact_directory(self):
         """Return the artifact directory of the bucket used to store image information."""
         return self._get_tag(PCLUSTER_S3_IMAGE_DIR_TAG)
 
+    @property
+    def tags(self):
+        """Return stack tags."""
+        return self._tags
```

Comment:
if we need tags outside we can avoid the underscore prefix and make the attribute private rather than having a property to expose the attribute.